app/static/js/adminManagement.js

```diff
@@ -43,56 +43,46 @@ function liveSearch(selectPickerID, e) {
-    document.getElementById("submitModal").setAttribute("value", "addAid");
-    $("#modal").modal("show");
-  }
-  else if (button == "add2" && $("#addSaasAdmin").val() != "") {
-    $("h2").html("SAAS Administrators");
-    $("p").html("Are you sure you want to add " + $("#addSaasAdmin option:selected").text() + " as a SAAS Administrator?");
-    document.getElementById("submitModal").setAttribute("name", "addSaas");
-    document.getElementById("submitModal").setAttribute("value", "addSaas");
-    $("#modal").modal("show");
-  }
-  else if (button == "remove" && $("#removelaborAdmin").val() != "") {
-    $("h2").html("Labor Administrators");
-    $("p").html("Are you sure you want to remove " + $("#removelaborAdmin option:selected").text() + " as a Labor Administrator?");
-    document.getElementById("submitModal").setAttribute("name", "remove");
-    document.getElementById("submitModal").setAttribute("value", "remove");
-    $("#modal").modal("show");
-  }
-  else if (button == "remove1" && $("#removeFinAidAdmin").val() != "") {
-    $("h2").html("Financial Aid Administrators");
-    $("p").html("Are you sure you want to remove " + $("#removeFinAidAdmin option:selected").text() + " as a Financial Aid Administrator?");
-    document.getElementById("submitModal").setAttribute("name", "removeAid");
-    document.getElementById("submitModal").setAttribute("value", "removeAid");
-    $("#modal").modal("show");
-  }
-  else if (button == "remove2" && $("#removeSaasAdmin").val() != "") {
-    $("h2").html("SAAS Administrators");
-    $("p").html("Are you sure you want to remove " + $("#removeSaasAdmin option:selected").text() + " as a SAAS Administrator?");
-    document.getElementById("submitModal").setAttribute("name", "removeSaas");
-    document.getElementById("submitModal").setAttribute("value", "removeSaas");
-    $("#modal").modal("show");
+  // map button IDs to {selectID, formID, header, actionText}
+  const config = {
+    add:   { select: "#addlaborAdmin",       form: "#laborAdminForm",   header: "Labor Administrators",       action: "add",    noun: "Labor Administrator" },
+    add1:  { select: "#addFinAidAdmin",      form: "#finAidAdminForm",  header: "Financial Aid Administrators", action: "add",    noun: "Financial Aid Administrator" },
+    add2:  { select: "#addSaasAdmin",        form: "#SAASAdminForm",    header: "SAAS Administrators",        action: "add",    noun: "SAAS Administrator" },
+    remove:{ select: "#removelaborAdmin",    form: "#laborAdminForm",   header: "Labor Administrators",       action: "remove", noun: "Labor Administrator" },
+    remove1:{ select: "#removeFinAidAdmin",  form: "#finAidAdminForm",  header: "Financial Aid Administrators", action: "remove", noun: "Financial Aid Administrator" },
+    remove2:{ select: "#removeSaasAdmin",    form: "#SAASAdminForm",    header: "SAAS Administrators",        action: "remove", noun: "SAAS Administrator" }
+  };
+
+  const cfg = config[button];
```

Comment:
`config` and `cfg` are too close to each other. let's come up with more descriptive names, particularly for `cfg`.